Dockerfile.template

```diff
@@ -240,10 +245,14 @@ RUN set -eux; \
 		| sort -u \
 		| xargs -r apt-mark manual \
 	; \
+# add cloud-init support 
```

Comment:
maybe this comment is better?
# add genisoimage for cloud-init support, form cloudutilssrc